aopy/data/bmi3d.py

```diff
@@ -2277,6 +2278,13 @@ def tabulate_stim_data(preproc_dir, subjects, ids, dates, metadata=['stimulation
     '''
     if df is None:
         df = pd.DataFrame()
+        
+    possible_triggers = ['qwalor_trigger', 'qwalor_ch1_trigger', 'qwalor_ch2_trigger', 
+                        'qwalor_ch3_trigger', 'qwalor_ch4_trigger']
+    possible_sensors = ['qwalor_sensor', 'qwalor_ch1_sensor', 'qwalor_ch2_sensor', 
+                       'qwalor_ch3_sensor', 'qwalor_ch4_sensor']
+    possible_stim_sites = ['stimulation_site', 'stimulation_site_ch1', 'stimulation_site_ch2',
+                           'stimulation_site_ch3', 'stimulation_site_ch4']
```

Comment:
Should this be loaded from an external datafile?

Reply:
yeah probably a good idea, i'll add something to the config folder